pages/desktop/frontend/search.py

```diff
@@ -111,6 +113,14 @@ def click_search_result(self, count):
 
             return Detail(self.selenium, self.page.base_url).wait_for_page_to_load()
 
+        def validate_filter_sort_by_rating(self):
+            for result in self.extensions:
+                assert getattr(result, 'rating') > 4
```

Comment:
Ok

Reply:
I think that if we just have a single comparison in this function and we return the result, the idea of having a separate function for this will only make the flow more complicate. I can just make the comparison inside the test and avoid using a function.